### PR TITLE
Updates to Sleep Number fetching

### PR DESCRIPTION
**Pull request by ClayBenson94**

### CODE REVIEW

File: asyncsleepiq/fuzion/sleeper.py [hunk 1]
```diff
@@ -22,9 +22,11 @@ async def set_sleepnumber(self, setting: int) -> None:
 
     async def fetch_sleepnumber(self) -> None:
         """Update fav_sleep_number from API."""
-        result = await self.api.bamkey(self.bed_id, "GetSleepNumberControls")
-        updating, current, last = result.split()
-        self.sleep_number = int(current)
+        args = [SIDES_FULL[self.side].lower()]
+        result = await self.api.bamkey(self.bed_id, "GetSleepNumberControls", args=args)
+        isUpdating, ambientNumber, userNumber = result.split()
+        print("CURRENT NUMBER FOR", self.side, result)
```
Comment (kbickar) [inline asyncsleepiq/fuzion/sleeper.py]: Remove prints

File: asyncsleepiq/fuzion/sleeper.py [hunk 2]
```diff
@@ -22,9 +22,11 @@ async def set_sleepnumber(self, setting: int) -> None:
 
     async def fetch_sleepnumber(self) -> None:
         """Update fav_sleep_number from API."""
-        result = await self.api.bamkey(self.bed_id, "GetSleepNumberControls")
-        updating, current, last = result.split()
-        self.sleep_number = int(current)
+        args = [SIDES_FULL[self.side].lower()]
+        result = await self.api.bamkey(self.bed_id, "GetSleepNumberControls", args=args)
+        isUpdating, ambientNumber, userNumber = result.split()
```
Comment (kbickar) [inline asyncsleepiq/fuzion/sleeper.py:27]: Should be snake_case in python

File: asyncsleepiq/fuzion/sleeper.py (hunk 1 above)
Comment (ClayBenson94) [reply asyncsleepiq/fuzion/sleeper.py]: My b - I removed in 48e50bee56e19fd27b2d0b6359abafa3cc63b599!

File: asyncsleepiq/fuzion/sleeper.py (hunk 2 above)
Comment (ClayBenson94) [reply asyncsleepiq/fuzion/sleeper.py]: 5830dc5

**Review by kbickar — APPROVED** [review]

**Pull request merged**